sormas-backend/src/main/java/de/symeda/sormas/backend/util/PatchHelper.java

```diff
@@ -0,0 +1,330 @@
+	 * @param <T>
+	 *            - the type of the object
+	 */
+	@Experimental
+	public static <T extends HasUuid> void postUpdate(@NotNull JsonNode jsonObject, @NotNull T existingObject) {
+
+		if (!existingObject.getUuid().equals(jsonObject.get(EntityDto.UUID).textValue())) {
+			throw new RuntimeException("The updated object must have a Uuid");
+		}
+		postUpdateOrReplace(jsonObject, existingObject);
+	}
+
+	private static <T extends HasUuid> void postUpdateOrReplace(@NotNull JsonNode jsonObject, @NotNull T existingObject) {
+
+		Iterator<Map.Entry<String, JsonNode>> jsonObjectFieldMap = jsonObject.fields();
+		while (jsonObjectFieldMap.hasNext()) {
+			Map.Entry<String, JsonNode> jsonObjectFieldMapEntry = jsonObjectFieldMap.next();
+			JsonNode jsonObjectFieldNode = jsonObjectFieldMapEntry.getValue();
+			Field existingObjectField = getClassFieldByName(jsonObjectFieldMapEntry.getKey(), existingObject.getClass());
+			existingObjectField.setAccessible(true);
+
+			if (jsonObjectFieldNode.isObject()) {
+				updateExistingObject(existingObject, jsonObjectFieldNode, existingObjectField);
+
+			} else if (jsonObjectFieldNode.isArray()) {
+				updateObjectList(existingObject, jsonObjectFieldNode, existingObjectField);
+
+			} else {
+				setFieldValue(existingObject, existingObjectField, jsonObjectFieldNode);
+			}
+		}
+	}
+
+	/**
+	 * This method updates the fields of type collections (lists or array).
+	 * 
+	 * @param existingObject
+	 * @param jsonObjectFieldNode
+	 * @param existingObjectField
+	 * @param <T>
+	 * @throws JsonProcessingException
+	 * @throws IllegalAccessException
+	 */
+	private static <T extends HasUuid> void updateObjectList(T existingObject, JsonNode jsonObjectFieldNode, Field existingObjectField) {
+		Class listElementClass = getParameterizedType(existingObjectField);
+		List tempNewElementList = new ArrayList();
+
+		for (JsonNode listElement : jsonObjectFieldNode) {
+			T updatedObjectList = addOrReplaceListElement(existingObject, existingObjectField, listElementClass, listElement);
+			if (updatedObjectList != null) {
+				tempNewElementList.add(updatedObjectList);
+			}
+		}
+
+		if (existingObjectField.getType().isAssignableFrom(List.class)) {
+			Object existingObjectFieldInstance = null;
+			try {
+				existingObjectFieldInstance = existingObjectField.get(existingObject);
+			} catch (IllegalAccessException e) {
+				throw new RuntimeException(
+					"Cannot access field " + existingObjectField.getName() + " on the object of type: " + existingObject.getClass().getName());
+			}
+			if (existingObjectFieldInstance != null) {
+				Collection existingElementList = (Collection) existingObjectFieldInstance;
+				existingElementList.clear();
+				existingElementList.addAll(tempNewElementList);
+			} else {
+				try {
+					existingObjectField.set(existingObject, tempNewElementList);
+				} catch (IllegalAccessException e) {
+					e.printStackTrace();
+					throw new RuntimeException(
```

Comment:
please use `RuntimeException(String message, Throwable cause)` instead of `e.printStackTrace(); throw new RuntimeException`